src/Murder.Editor/Data/EditorDataManager.cs

```diff
@@ -193,11 +193,12 @@ public override void RefreshAtlas()
 
         private void ScanHighResImages()
         {
-            if (!Directory.Exists(EditorSettings.RawResourcesPath))
+            string fullRawResourcesPath = FileHelper.GetPath(EditorSettings.RawResourcesPath);
+            if (!Directory.Exists(fullRawResourcesPath))
             {
-                GameLogger.Log($"Unable to find raw resources path at {FileHelper.GetPath(EditorSettings.RawResourcesPath)}. " +
+                GameLogger.Log($"Unable to find raw resources path at {FileHelper.GetPath(fullRawResourcesPath)}. " +
```

Comment:
```suggestion
                GameLogger.Log($"Unable to find raw resources path at {fullRawResourcesPath}. " +
```

since that's already the full path